docs/src/components/typography/SectionHeadline.tsx

```diff
@@ -2,20 +2,33 @@ import * as React from 'react';
 import { useTheme } from '@mui/material/styles';
 import Typography from '@mui/material/Typography';
 
+interface SectionHeadlineTheme {
+  overlineColor?: string;
+  titleColor?: string;
+  descriptionColor?: string;
+}
+
 export default function SectionHeadline({
   overline,
   title,
   description,
+  localTheme,
 }: {
   overline: React.ReactNode;
   title: React.ReactNode;
   description?: React.ReactNode;
+  localTheme?: (mode?: string) => SectionHeadlineTheme;
```

Comment:
This is to be able to customise the section headlines, since the fill colour in https://www.figma.com/file/fNowz3HeGrUJshzjUvvY1e/Toolpad-landing-page?node-id=1019%3A64196 is different from the default component

Reply:
Are we talking about "Feature" that is supposed to be primary.500 but is currently `primary.600`? 

<img width="494" alt="Screenshot 2022-08-21 at 01 55 16" src="https://user-images.githubusercontent.com/3165635/185769778-334853c2-88be-480f-8d11-d6e9ef971194.png">

I see primary 500 too in https://www.figma.com/file/4uv2dT18rXJPZBbrbpw61q/Marketing-site?node-id=3224%3A11442. I believe it's a tweak that we did later on, to improve the design, that it's Figma that is outdated, so 

```suggestion
```

Reply:
The existing colour in the component is:
`const defaultOverlineColor = mode === 'dark' ? 'primary.300' : 'primary.600';`

whereas, the Figma file has a `primary.500` set in both light and dark cases

Reply:
Nevertheless,  I have removed this local override

Reply:
ah. thank you! I was indeed following the convention in Figma.

Reply:
Ok, so we are good, the configutaton option change was off because there is no reason for the Toolpad landing page to have a different look than the other pages. it's either we change them all, or we consider it's already correct.